applications/GeoMechanicsApplication/tests/settlement_workflow.py

```diff
@@ -35,9 +35,28 @@ def test_d_settlement_workflow(self):
                 "DISPLACEMENT", times_to_check[i], expected_data, node_ids)
 
             self.assertEqual(len(actual_nodal_values), len(expected_nodal_values))
-            for (actual_displacement, expected_displacement) in zip(actual_nodal_values, expected_nodal_values):
+            for actual_displacement, expected_displacement in zip(actual_nodal_values, expected_nodal_values):
                 self.assertVectorAlmostEqual(actual_displacement, expected_displacement, 3)
 
+            if i > 2:
+                self.check_stress_values(expected_result_file_name, i, node_ids, reader, result_file_name,
+                                         times_to_check, "TOTAL_STRESS_TENSOR")
+                self.check_stress_values(expected_result_file_name, i, node_ids, reader, result_file_name,
+                                         times_to_check, "CAUCHY_STRESS_TENSOR")
+
+    def check_stress_values(self, expected_result_file_name, i, node_ids, reader, result_file_name, times_to_check,
```

Comment:
Since `times_to_check` and `i` are only used here to get hold of the time, we might as well just pass the `time` (so replace the former two parameters by the latter one). This makes the method simpler.

Reply:
Good point, changed!